Description: make the generated .abi.tar.gz files reproducible
Forwarded: not-needed
Bug-Debian: http://bugs.debian.org/798481
Author: Chris Lamb <lamby@debian.org>
Reviewed-by: Mathieu Malaterre <malat@debian.org>

Index: abi-compliance-checker-2.3/abi-compliance-checker.pl
===================================================================
--- abi-compliance-checker-2.3.orig/abi-compliance-checker.pl
+++ abi-compliance-checker-2.3/abi-compliance-checker.pl
@@ -9529,6 +9529,10 @@
         open(DUMP, ">", $DPath) || die ("can't open file \'$DPath\': $!\n");
         print DUMP $ABI_DUMP;
         close(DUMP);
+
+        # Ensure contents of archive have deterministic contents
+        chmod(0644, $DPath);
+        utime(0, 0, $DPath);
         
         if(not -s $DPath) {
             exitStatus("Error", "can't create ABI dump because something is going wrong with the Data::Dumper module");
